### PR TITLE
Remove AWS and Terraform resources.

### PR DESCRIPTION
**Pull request by flamingbear**
DAS-1761

## Description

Deletes all AWS references for testing, tests no longer can be automatically
deployed to AWS.  There was at one time a reason to test in us-west-2, but no
one can remember what that reason is.  

## Jira Issue ID

[DAS-1747](https://bugs.earthdata.nasa.gov/browse/DAS-1747)

## Local Test Steps

Check out this branch and follow the instructions for test from the previous PR #53 

## PR Acceptance Checklist
* [ ] Acceptance criteria met
* [ ] Tests added/updated (if needed) and passing
* [ ] Documentation updated (if needed)

### CODE REVIEW

**Comment by chris-durbin** [comment]
In order to access some of the data for the tests, I thought we needed to run from within certain AWS accounts. I also thought for some of the zarr verifications we used S3 URLs as well that can only be used in region. @indiejames might remember other reasons and whether they are all obsolete.

**Comment by flamingbear** [comment]
> In order to access some of the data for the tests, I thought we needed to run from within certain AWS accounts. I also thought for some of the zarr verifications we used S3 URLs as well that can only be used in region. @indiejames might remember other reasons and whether they are all obsolete.

We do download data from s3 during the zarr test, but the credentials in bamboo should be enough to get that data.  None of the other tests need to access data outside of the sample data in this repository.  So the regressions can and are run from local desktops and soon from a bamboo server.

**Comment by indiejames** [comment]
The only thing I (barely) remember was that there were certain cells that would only run correctly from us-west-2. I don't remember any of the details, tough, so maybe that is no loner true.

File: README.md [hunk 1]
```diff
@@ -31,19 +27,19 @@ From the `./test` directory make all of the regression images with:
     $ make images
 
 *`make -j images` can be used to make the images in parallel (faster), although this may lead to
-Docker Desktop*
+Docker Desktop instabilities*
```
Comment (owenlittlejohns) [inline README.md:30]: Interesting. I thought this line used to include this. Ah well. Good catch!

File: README.md [hunk 2]
```diff
@@ -60,7 +56,7 @@ created Zarr store.*
 1. *`HARMONY_HOST_URL` is the harmony base url for your target environment. e.g. `SIT` would be `https://harmony.sit.earthdata.nasa.gov`*
 
 
-### test in a Browswer:
+### Test in a Browser:
```
Comment (owenlittlejohns) [inline README.md:59]: Ha - I totally saw this show up tangentially in your last PR (in one of the lines near a change you made) and meant to fix it. Nice catch.

**Review by owenlittlejohns — APPROVED** [review]
Thanks for the clean-up. Looks good to me.

**Comment by owenlittlejohns** [comment]
@chris-durbin - just saw your comments above. I have tested changes to the Bamboo build plan that run using the new `test-in-bamboo.sh` script (pulling Docker images from ghcr.io and not using Terraform resources). The tests all ran and passed.

When we did the first half of this epic (publishing the versioned Docker images for each test suite to ghcr.io), we verified that each test suite ran locally using the `run_notebooks.sh` script also without those Terraform resources. I think we could hit complications if we wanted a regression test for a large output for something like NetCDF-to-Zarr, but the sample aggregation request was kept small, and the store is pulled from S3 to the Docker container at runtime.

File: README.md (hunk 1 above)
Comment (flamingbear) [reply README.md:30]: It did and I noticed it was missing. /shrug

**Comment by flamingbear** [comment]
@chris-durbin & @indiejames any concerns remaining here?

**Review by chris-durbin — APPROVED** [review]

**Pull request merged**